src/wasm-lib/kcl/src/std/math.rs

```diff
@@ -39,8 +42,13 @@ pub async fn rem(_exec_state: &mut ExecState, args: Args) -> Result<KclValue, Kc
         divisor = "The number which will divide `num`.",
     }
 }]
-fn inner_rem(num: i64, divisor: i64) -> Result<i64, KclError> {
-    Ok(num % divisor)
+fn inner_rem(num: NumberArg, divisor: NumberArg) -> NumberArg {
+    match (num, divisor) {
+        // Possible implicit conversion from int to float.
+        (NumberArg::Float(_), _) | (_, NumberArg::Float(_)) => NumberArg::Float(num.to_f64() % divisor.to_f64()),
```

Comment:
What're the scenarios where the implicit conversion bites someone in the butt?

Reply:
I don't know. With the deprecation of KCL's `int()` function, I think it's expected to happen by users.